kube-monitoring/charts/templates/_alertmanager-config.yaml.tpl

```diff
@@ -2,8 +2,13 @@
   scheme: https
 {{- if and .Values.alerts.enabled .Values.alerts.alertmanagers.hosts }}
   tls_config:
-    cert_file: /etc/prometheus/secrets/tls-prometheus-alertmanager-auth/tls.crt
-    key_file: /etc/prometheus/secrets/tls-prometheus-alertmanager-auth/tls.key
+{{- if and .Values.alerts.alertmanagers.tlsConfig.cert .Values.alerts.alertmanagers.tlsConfig.key }} 
+    cert_file: /etc/prometheus/secrets/tls-prometheus-{{ .Release.Name }}/tls.crt
+    key_file: /etc/prometheus/secrets/tls-prometheus-{{ .Release.Name }}/tls.key
+{{- else }}
+    cert_file: /etc/prometheus/secrets/tls-prometheus-{{ .Release.Namespace }}/tls.crt
+    key_file: /etc/prometheus/secrets/tls-prometheus-{{ .Release.Namespace }}/tls.key
+{{- end }}
```

Comment:
This would always point to the dummy-secret, wouldn't it?

In this case nothing would work, but the at least the operator wouldn't complain. This captures the case of a prometheus without an alertmanager then, is this something we need?

Reply:
The `dummySecret` is just for the github-action as the alerts plugin is a dependency.